ShowMyData/Pages/Index.cshtml.cs

```diff
@@ -7,35 +7,52 @@
 using System.Net;
 using Crime;
 using Drugs;
+using Newtonsoft.Json.Schema;
+using Newtonsoft.Json.Linq;
 
 namespace ShowMyData.Pages
 {
     public class IndexModel : PageModel
     {
         public void OnGet()
         {
-            SearchCompleted = false;
+            searchCompleted = false;
         }
         [BindProperty]
-        public string Search { get; set; }
-        public bool SearchCompleted { get; set; }
+        public string search { get; set; }
+        public bool searchCompleted { get; set; }
```

Comment:
Actually... in this syntax, these variables are considered wrappers around getter and setter methods.  Since methods are PascalCase in .net, these variables are often PascalCase as well.  Might as well keep things as they are.